CHANGELOG.md

```diff
@@ -7,81 +7,87 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
-## [0.2.2] - 2019-08-15
```

Comment:
@craicoverflow 

We use the format defined here: https://keepachangelog.com/en/1.0.0/

I see you noticed some problems in the file and fixed them, but there's just one minor problem. The versions should start with 2 `#` chars.

It should be like
```
# Changelog
...

## [Unreleased]
 <Nothing here>

## [0.2.3] - 2019-08-23
### Changed
- Updaed version of Mobile Developer Console

## [0.2.2] - 2019-08-15
...
```

Reply:
Ah thanks, I will fix that.

Reply:
👍